drf_spectacular/templates/drf_spectacular/swagger_ui.html

```diff
@@ -4,26 +4,28 @@
     <title>Swagger</title>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" href="//unpkg.com/swagger-ui-dist@3.35.0/swagger-ui.css" />
+    <link rel="icon" type="image/png" href="{{swagger_bundle_favicon_href}}"/>
```

Comment:
not sure about the favicon. if people serve their company favicon on their domain, this would override it.

Reply:
what about these way? 

~~~Python

# SPECTACULAR_DEFAULTS

{
  ....
  'FAVICON_HREF': '//unpkg.com/swagger-ui-dist@3.35.1/favicon-32x32.png', # default favicon href
  ...  
}

~~~

this is just nitpick.  

if you do not sure about tihs, I will rollback

Reply:
sure we can do this. just one comment. wrap it in a template `if`. if `favicon_href` is empty/falsy, don't include the `link` tag. that way people can turn it of easily in the settings.